synthtool/gcp/common.py

```diff
@@ -358,3 +371,9 @@ def _load_repo_metadata(metadata_file: str = "./.repo-metadata.json") -> Dict:
         with open(metadata_file) as f:
             return json.load(f)
     return {}
+
+
+# Returns the output of `python foo.py --help`
+def _get_help(filename: str) -> str:
+    """Function used by sample readmegen"""
+    return shell.run(["python", filename, "--help"]).stdout
```

Comment:
After further review, I'd rather see this one line of code be a lambda on line 63. Is there an advantage of breaking it out into a separate function?

Reply:
nope sounds good, will refactor

Reply:
updated!